kafka-rest/src/main/java/io/confluent/kafkarest/entities/ConsumerGroup.java

```diff
@@ -42,6 +45,16 @@
 
   public abstract ImmutableList<Consumer> getConsumers();
 
+  public final ImmutableMap<Partition, Consumer> getPartitionAssignment() {
+    Map<Partition, Consumer> partitionAssignment = new HashMap<>();
```

Comment:
You can use `ImmutableMap.Builder` here if you want to avoid the cost of copying the map later.